test/unit/generator/content/schedule/part3/get-table.test.js

```diff
@@ -1,4 +1,5 @@
-const { getTable } = require('../../../../../../app/generator/content/schedule/part1/get-table')
+const { getTable } = require('../../../../../../app/generator/content/schedule/part3/get-table')
+const toCurrencyString = require('../../../../../../app/generator/to-currency-string')
 let mockSchedule
 
 describe('get table', () => {
```

Comment:
Can we get these tests again for a reduction schedule?

Reply:
Added